test/util/appendingDirectOnly.js

```diff
@@ -0,0 +1,185 @@
+module.exports = fallbackDependancySandBox
+
+function fallbackDependancySandBox (appDir) {
```

Comment:
appDir is never used for anything. we can remove all the logic that depends on it